client/sagas/build-challenge-epic.js

```diff
@@ -38,6 +42,43 @@ export default function buildChallengeEpic(actions, getState) {
           .map(frameTests)
           .startWith(initOutput('// running test'));
       }
+      /* this is a React Challenge: */
+      if (challengeNumber === 8) {
```

Comment:
We don't want to add paths specify to challenge types. The next iteration (modern) should handle everything without requiring specific paths. Otherwise adding new curriculum will get out of hand quickly.